src/components/page-layout/page-layout.js

```diff
@@ -101,9 +76,6 @@ export default class PageLayout extends React.Component {
     return (
       <ErrorBoundary>
         {!noShellHeaderBuffer && <div className="shell-header-buffer" />}
-        <ErrorBoundary>
-          {this.renderTopbar(switchedNavigation, parentPath)}
-        </ErrorBoundary>
         <div className="limiter">
```

Comment:
We may want to create our own limiter class -- perhaps one with a max-width of 1800 pixels.

Reply:
I spent a few minutes testing this out and made a horrible mess of things, so decided to leave it for now.